### PR TITLE
updated README.md for day 3 & 4, uploaded tickets for day 3 & 4

### PR DESCRIPTION
**Pull request by yuichiu416**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,5 +90,29 @@ information.
 * 5:00 PM	Go for Dinner
 * 8:00 PM	Go back to hotel
 
+### Day 3 (2018/3/26)
+#### Key Words: [Surfing](tickets/surfing.jpg) and[ Polynesian Culture Center trip](tickets/Polynesian_Culture_Center.jpg)
+* 7:00 AM	Breakfast in AirBnB House
+* 8:30 AM	Heading to beach
+* 9:0 AM	Surfing
```
Comment (Scholars-Mate) [inline README.md]: Forgot a zero here

File: README.md [hunk 2]
```diff
@@ -90,5 +90,29 @@ information.
...
+* 11:00 AM  Grabing some food and heading to Polynesian Culture Center
+* 12:00 PM Polynesian Culture Center tour
+* 6:00 PM   Dinner at Polynesian Culture Center
+* 9:00 PM	Go back to hotel
```
Comment (Scholars-Mate) [inline README.md]: Choose either spaces or tabs. Try not to mix them

**Review by Scholars-Mate — CHANGES_REQUESTED** [review]
Just a few quick changes.

**Comment by yuichiu416** [comment]
I don't know what happened, lol.
I fixed the indent and pushed it but I didn't see anything on the website, should I close this one and request another pull request?

**Review by Scholars-Mate — APPROVED** [review]

**Pull request merged**